src/dmd/astbase.d

```diff
@@ -3933,21 +3933,33 @@ struct ASTBase
 
     extern (C++) class TypeFunction : TypeNext
     {
-        ParameterList parameterList;  // function parameters
-
-        bool isnothrow;             // true: nothrow
-        bool isnogc;                // true: is @nogc
-        bool isproperty;            // can be called without parentheses
-        bool isref;                 // true: returns a reference
-        bool isreturn;              // true: 'this' is returned by ref
-        bool isscope;               // true: 'this' is scope
-        bool islive;                // true: function is @live
+        // .next is the return type
+
+        ParameterList parameterList;   // function parameters
+
+        private enum FunctionFlag
+        {
+            isnothrow       = 0x0001, // nothrow
```

Comment:
Might as well add a `/` to turn those into DDoc comments.

Reply:
it's private and it's just a copy of the original code comments. better DDOC is in the public accessors.

Reply:
`private` doesn't really matter IMHO because DDoc comments are shown in IDE's anyway.

But i guess it  doesn't matter as much in this case.